client/src/common/components/Home.tsx

```diff
@@ -81,10 +80,16 @@ const Home = () => {
 					className="absolute w-48 h-48 md:w-72 md:h-72 xl:w-full xl:h-full top-1/4 -right-14 xl:-top-0 xl:-right-1/2"
 				/>
 			</section>
-			<section className="p-10 md:p-20">
-				<h2 className="text-2xl font-bold text-center md:text-5xl">Q&As</h2>
-				<Accordion className="p-10 text-sm md:text-2xl w-auto">
-					<AccordionItem className="p-5">
+			<section className="p-10 md:p-20 relative">
+				<h2 className="text-2xl md:text-5xl text-center font-bold mb-10 md:mb-20">
+					Q&As
+				</h2>
+				<Accordion
+					allowMultipleExpanded
+					allowZeroExpanded
```

Comment:
@Dannyboy222 these props are the ones you wanted to add.

Reply:
thank you!